UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/BatterySystem.cs

```diff
@@ -92,7 +92,7 @@ public float ChargePercentage()
 			{
 				foreach (var Cell in BatteryPack.Cells)
 				{
-					charge += Cell.Watts;
+					charge += Cell.watts;
```

Comment:
?????
why is it lower case, this looks like a public variable; it should stick to our conventions and make it capital